ekncontent/Makefile.am

```diff
@@ -244,7 +244,7 @@ typelibdir = $(libdir)/girepository-1.0
 typelib_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
 
 overridesdir = $(datadir)/gjs-1.0/overrides
-overrides_DATA = overrides/EosKnowledgeContent.js
```

Comment:
Contingent on whether the node bindings move to another repo or not. (If they do, then I'd prefer to keep `overrides` since that mirrors the directory where they have to be installed.)